Lib/test/test_pty.py

```diff
@@ -166,10 +167,12 @@ def test_openpty(self):
         if self.stdin_rows != None and self.stdin_cols != None:
             try:
                 debug("Setting pty.STDIN_FILENO window size")
-                # Set number of columns and rows to be the
-                # floors of 1/5 of respective original values
-                target_stdin_winsz = struct.pack("HHHH", self.stdin_rows//5,
-                                                 self.stdin_cols//5, 0, 0)
+                debug(f"original size: (rows={self.stdin_rows}, cols={self.stdin_cols})")
+                target_stdin_rows = self.stdin_rows + 1
```

Comment:
Please add a few words to describe the resize trick and why it is necessary.

Reply:
Does this look good?

Reply:
I have added some comments. Please let me know if this looks good.